docs/scos/dev/front-end-development/202212.0/oryx/reactivity/signals.md

```diff
@@ -0,0 +1,107 @@
+# Oryx Framework - Signals
+
+## Overview
+
+Signals are crucial tools within the Oryx framework, offering a clean and efficient reactivity API for component design. They allow for the declarative formulation of component logic while seamlessly integrating with observables from domain services.
+
+## Implementation of Signals in Oryx
+
+The Oryx implementation of signals has a core mechanism and a simplified API. The core is well-suited for advanced usage, while the simplified API is sufficiently robust for most components. This documentation will focus on the simplified API.
+
+### Creating Signals
+
+To create a signal, use the `signal()` function. This function can either take a raw value or accept an observable.
+
+Here's an example of creating a simple signal:
+
+```ts
+const counter = signal(1);
+```
+
+Changing the signal value:
+
+```ts
+counter.set(2);
+```
+
+Creating a signal from an observable:
+
+```ts
+const values = signal(observable$);
+```
+
+#### Signal Options
+
+Signal options are settings that you can use to customize your signals.
+
+- `equal` function: This option allows you to compare two values and decide if they are equal. It's a function that takes two inputs and should return true if they are equal and false if they're not.
```

Comment:
can you add small explanation why this is needed?

Reply:
👍